cfme/tests/intelligence/test_optimization.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from widgetastic.utils import attributize_string
+
+from cfme import test_requirements
+from cfme.utils.blockers import BZ
+
+pytestmark = [test_requirements.report, pytest.mark.tier(1), pytest.mark.ignore_stream("5.10")]
+
+REPORTS = [
+    "Host CPU Trends (last week)",
+    "Host Memory Trends (last week)",
+    "Offline VMs with Snapshot",
+    "Top CPU Consumers (weekly)",
+    "Top Memory Consumers (weekly)",
+    "VMs with Volume Free Space >= 75%",
+]
+
+
+@pytest.mark.meta(automates=[1769346])
+@pytest.mark.uncollectif(
+    lambda menu_name: menu_name in ["Top CPU Consumers (weekly)", "Top Memory Consumers (weekly)"]
+    and BZ(1769346, forced_streams=["5.11"]).blocks,
+    reason="Breadcrumb for these reports is incorrect which is why the exists method fails.",
```

Comment:
:tada:

Reply:
This _might_ be better defined as a blocker meta, with an unblock lambda. Then the BZ.blocks is implicit.

Reply:
Defining as a blocker meta would provide more accurate test result metrics as well - we would see that there are two broken test cases in the test result (skipped via BZ) versus with this, we are just missing two test cases that _should_ be covered.

Reply:
Using `pytest.skip` instead.